### PR TITLE
[8306] delete more apps and drop a4 tables

### PR DESCRIPTION
**Pull request by goapunk**
- delete all apps which had migrations deleting their models in the previous PR
- manual migration to drop all a4 tables
- prepare removal of organisation app
- reset migrations for home app as they depend on other migrations (see commit on how to fix migration state manually)

**This PR needs to be merged after https://github.com/liqd/a4-opin/pull/2689 was deployed!**

### CODE REVIEW

File: euth/contrib/migrations/0001_delete_a4_models.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+    ]
+
+    operations = [
+        migrations.RunSQL(
+            sql=[
+                "DROP TABLE IF EXISTS background_task",
```
Comment (m4ra) [inline euth/contrib/migrations/0001_delete_a4_models.py:14]: hmm this didn't work for background_task table. Did it work for you @goapunk?

Comment (goapunk) [reply euth/contrib/migrations/0001_delete_a4_models.py:14]: will double check, I copied it from mb/a+ though so i assume it should have worked

Comment (goapunk) [reply euth/contrib/migrations/0001_delete_a4_models.py:14]: ah, I forgot to remove the package from the installed apps and dependencies

**Comment by goapunk** [comment]
@m4ra this should hopefully be ready again

**Review by m4ra — APPROVED** [review]
thanks!

**Pull request merged**